distributed/utils.py

```diff
@@ -205,26 +206,14 @@ def All(*args):
     raise gen.Return(results)
 
 
-def is_loop_running(loop):
-    """
-    Return whether an IOLoop is running.
-    """
-    # Tornado < 5.0
-    r = getattr(loop, '_running', None)
-    if r is not None:
-        return r
-    try:
-        # Tornado 5.0 with AsyncIOLoop (default setting)
-        return loop.asyncio_loop.is_running()
-    except AttributeError:
-        raise TypeError("don't know how to query the running state of %s"
-                        % (type(loop),))
-
-
 def sync(loop, func, *args, **kwargs):
     """
     Run coroutine in loop running in separate thread.
     """
+    # Tornado's PollIOLoop doesn't raise when using closed, do it ourselves
+    if isinstance(loop, PollIOLoop) and getattr(loop, '_closing', False):
+        raise RuntimeError("IOLoop is closed")
```

Comment:
Why was this necessary?  Who uses `PollIOLoop`?  Windows?

Reply:
Python 2 on Tornado 5, and all users of Tornado 4 :-)

Reply:
Ah, indeed

```python
In [1]: from tornado.ioloop import IOLoop

In [2]: type(IOLoop.current()).mro()
Out[2]: 
[tornado.platform.epoll.EPollIOLoop,
 tornado.ioloop.PollIOLoop,
 tornado.ioloop.IOLoop,
 tornado.util.Configurable,
 object]
```

I was on master when I wrote that comment